packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/api/DiagramImageConstants.java

```diff
@@ -19,6 +19,8 @@
  */
 public final class DiagramImageConstants {
 
+    public static final String HIDE_SVG = "/icons/full/obj16/HideTool.svg";
```

Comment:
Where is this image coming from? Do not add coupling between unrelated modules. Is that coming from the diagram view DSL? If so, `sirius-components-collaborative-diagrams` can be reused with the view DSL so it cannot be referenced this way. Since I don't see any reference to this constant anywhere in `sirius-components-collaborative-diagrams`, you can remove it entirely